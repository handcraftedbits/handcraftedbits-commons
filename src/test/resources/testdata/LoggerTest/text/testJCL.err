WARN  | main                 | c.h.c.api.log.LoggerTest  | warn from JCL 
WARN  | main                 | c.h.c.api.log.LoggerTest  | warn from JCL with MDC | key1=value1, key2=value2
ERROR | main                 | c.h.c.api.log.LoggerTest  | error from JCL 
ERROR | main                 | c.h.c.api.log.LoggerTest  | error from JCL with MDC | key1=value1, key2=value2
