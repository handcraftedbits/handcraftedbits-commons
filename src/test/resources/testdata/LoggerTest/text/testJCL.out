TRACE | main                 | c.h.c.api.log.LoggerTest  | trace from JCL 
TRACE | main                 | c.h.c.api.log.LoggerTest  | trace from JCL with MDC | key1=value1, key2=value2
DEBUG | main                 | c.h.c.api.log.LoggerTest  | debug from JCL 
DEBUG | main                 | c.h.c.api.log.LoggerTest  | debug from JCL with MDC | key1=value1, key2=value2
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from JCL 
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from JCL with MDC | key1=value1, key2=value2
